lib/tableau_api/resources/workbooks.rb

```diff
@@ -92,7 +95,7 @@ def add_permissions(workbook_id:, user_id: nil, group_id: nil, capabilities:)
         res.code == 200
       end
 
-      def delete_permissions(workbook_id:, user_id: nil, group_id: nil, capability:, capability_mode:)
+      def delete_permissions(workbook_id:, capability:, capability_mode:, user_id: nil, group_id: nil)
```

Comment:
I have minor concerns about the shuffling of parameters here and on `add_permissions`.  I see the specs explicitly name parameters when calling these methods but I worry this will affect users of this gem. Did you do a review of references of these methods in Console/Platform?

Reply:
I don't think order matters for ruby keyword arguments!

Reply:
good call- this was a rubocop complaint fix. I'm trying to set up console testing right now, but if this turns out to be a problem it'll get rubocop-ignored. I do believe ruby is not very strict about order of keyword args, so I'm hopeful...

Reply:
Cool. I think awareness is all that's needed. Good to go on this one.